backend/src/openarchiefbeheer/destruction/tests/test_models.py

```diff
@@ -686,3 +688,36 @@ def test_clean_local_metadata(self):
 
         self.assertEqual(item1.extra_zaak_data, {})
         self.assertEqual(item2.extra_zaak_data, {})
+
+
+class DestructionListCoReviewTest(TestCase):
+    def test_create(self):
+        DestructionListCoReviewFactory.create()
+
+    def test_created(self):
+        co_review = DestructionListCoReviewFactory.create()
+        self.assertTrue(co_review.created)
```

Comment:
I think that test_create can be removed and the check in test_created can be moved to the one below?

Reply:
- [ ] refactor test